src/Framework/UnitTests/EventArgs_Tests.cs

```diff
@@ -61,8 +61,7 @@ public void EventArgsCtors()
         /// <summary>
         /// Verify a whidbey project started event can be deserialized, the whidbey event is stored in a serialized base64 string.
         /// </summary>
-        [Fact(Skip = "Ignored in MSTest")]
-        // Ignore: Type in serialized string targets MSBuild retail public key, will not de-serialize
```

Comment:
Apparently the public key change wasn't the *only* reason this test was broken. Didn't dawn on me at the time :(